lib/src/androidTest/kotlin/at/bitfire/ical4android/AndroidEventTest.kt

```diff
@@ -1453,6 +1454,7 @@ class AndroidEventTest {
             destinationCalendar: TestCalendar = calendar,
             asSyncAdapter: Boolean = false,
             insertCallback: (id: Long) -> Unit = {},
+            extendedProperties: Map<String, String> = emptyMap(),
             valuesBuilder: ContentValues.() -> Unit
```

Comment:
We can set this to `= {}` for the case that we only want to test only an extended property (and remove all `valuesBuilder = {}`).